### PR TITLE
WALL-33: Create PropertyCard comp

### PR DESCRIPTION
**Pull request by manikangkan**
## Summary

Create PropertyCard comp

### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (code change that neither fixes a bug nor adds a feature)

### Linked Issues

Null

## JIRA Issue(s)

- [JIRA Issue Link](https://wallaceproject.atlassian.net/browse/WALL-32)

## Checklist (check all)

- [x] Automated tests have been added, or an explanation for why not
- [ ] Security implications and dependencies on linked features have been considered and explained

## Screenshots & Evidence
image
image
image
image

### CODE REVIEW

File: src/modules/PropertyCard/PropertyCard.tsx [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+              size="lg"
+              weight="medium"
+              className="line-clamp-1"
+              color={isBgDark ? 'inverted' : 'black'}
+            >
+              {price}
+            </Body>
+          </div>
+
+          {amenities.length > 0 && (
```
Comment (raj-edition) [inline src/modules/PropertyCard/PropertyCard.tsx:138]: lets create a common utility function for arrays that check if the array is empty or not. Lets use that at all places moving forward

**Review by raj-edition — COMMENTED** [review]
One change Mani

File: src/modules/PropertyCard/PropertyCard.tsx [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+                height="40"
+                viewBox="0 0 21 40"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg"
+                aria-hidden="true"
+              >
+                <path d="M21 39.9994L21 0.000244141L20.739 0.000505654L0 39.9996L21 39.9994Z" />
+              </svg>
+            </Body>
+          )}
```
Comment (raj-edition) [inline src/modules/PropertyCard/PropertyCard.tsx:112]: update to use https://github.com/wallace-developments/wallace-frontend/pull/21

**Review by raj-edition — CHANGES_REQUESTED** [review]

**Review by VasudevaVellanki4 — APPROVED** [review]

File: src/modules/PropertyCard/PropertyCard.tsx (hunk 1 above)
Comment (manikangkan) [reply src/modules/PropertyCard/PropertyCard.tsx]: Sorted

File: src/modules/PropertyCard/PropertyCard.tsx (hunk 2 above)
Comment (manikangkan) [reply src/modules/PropertyCard/PropertyCard.tsx]: Sorted

**Review by VasudevaVellanki4 — APPROVED** [review]

**Review by raj-edition — APPROVED** [review]

**Pull request merged**